src/wallet/spend.cpp

```diff
@@ -583,12 +583,13 @@ static bool IsCurrentForAntiFeeSniping(interfaces::Chain& chain, const uint256&
 }
 
 /**
- * Return a height-based locktime for new transactions (uses the height of the
+ * Set a height-based locktime for new transactions (uses the height of the
  * current chain tip unless we are not synced with the current chain
  */
-static uint32_t GetLocktimeForNewTransaction(interfaces::Chain& chain, const uint256& block_hash, int block_height)
+static void AntiFeeSnipe(CMutableTransaction& tx, interfaces::Chain& chain, const uint256& block_hash, int block_height)
```

Comment:
nit: I'd probably add a verb to the function's name

Reply:
Isn't `to snipe` a verb? Though, I am open to suggestions.

Reply:
I thought fee sniping is that something we want to counter, not something we're doing ourselves. I was thinking along the lines of "SetAntiFeeSnipe" or "SetAntiFeeSnipeLocktime" 🤷‍♂️

Reply:
Thx, done something